pandas/core/ops/__init__.py

```diff
@@ -312,7 +315,7 @@ def to_series(right):
 
 
 def should_reindex_frame_op(
-    left: DataFrame, right, op, axis, default_axis, fill_value, level
+    left: DataFrame, right, op, axis: int, fill_value, level
```

Comment:
Not a big deal but I guess we have AxisInt in pandas._typing for this already

Reply:
will update in next pass, dont want to futher clog the CI